app/containers/BalanceContainer/index.js

```diff
@@ -0,0 +1,78 @@
+/*
+ *
+ * BalanceContainer
+ *
+ */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import Helmet from 'react-helmet';
+import selectBalanceContainer from './selectors';
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+import styles from './styles.css';
+import BalanceCell from 'components/BalanceCell';
+import $ from 'jquery'; 
+import BigNumber from 'bignumber.js';
+
+export class BalanceContainer extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  constructor(props) {    
+    super(props);
+
+    this.state = {
+      balance: 0,
+      name: ""
+    };
+  }
+
+  componentDidMount() {
+    if (this.props.token == null) {
```

Comment:
`==` is a less specific equality operator. Default to `===`.
`null` is a falsey value. A cleaner way to write this in JS would be `if (!this.props.token) {...}`.